src/main/java/core/db/DBUpdater.java

```diff
@@ -115,6 +115,10 @@ void updateDB(int DBVersion) {
 	private void updateDBv301(int dbVersion, int version) {
 
 		try {
+
+			m_clJDBCAdapter.executeUpdate("ALTER TABLE MATCHESKURZINFO ALTER COLUMN isDerby SET DATA TYPE BOOLEAN");
```

Comment:
@tychobrailleur I was not sure if we need to check the type of the column first of if we could do this directly (I checked and it does not throws an error if called on integer columns)

Reply:
I don't think you need to check the type first, no.  To be sure I did a quick test too, and that's fine.

It works with integer to boolean, that's the main thing, because it's narrowing the column size.

Reply:
ok cool thank you